programs/drift/src/math/oracle.rs

```diff
@@ -252,12 +279,19 @@ pub fn oracle_validity(
         .confidence_interval_max_size
         .safe_mul(max_confidence_interval_multiplier)?);
 
-    let is_stale_for_amm = if slots_before_stale_for_amm_override != 0 {
-        oracle_delay.gt(&slots_before_stale_for_amm_override.max(0).cast()?)
+    let is_stale_for_amm_immediate = if slots_before_stale_for_amm_immdiate_override != 0 {
+        oracle_delay.gt(&slots_before_stale_for_amm_immdiate_override.max(0).cast()?)
     } else {
         oracle_delay.gt(&valid_oracle_guard_rails.slots_before_stale_for_amm)
     };
 
+    let is_stale_for_amm_low_risk = if taker_speed_bump_override != 0 {
+        oracle_delay.gt(&taker_speed_bump_override.max(0).cast()?)
+    } else {
+        // Default to current behavior if not set on the amm
+        false
```

Comment:
isnt default `true` currently? also, i think it's better if this one compares to `slots_before_stale_for_amm_immdiate_override` 🤔